spec/features/users/sign_in_spec.rb

```diff
@@ -92,10 +92,22 @@
   end
 
   context 'signed out' do
+    it 'keeps the user on the current page after session times out', js: true do
+      allow(Devise).to receive(:timeout_in).and_return(0)
+
+      visit new_user_registration_path
+      fill_in 'Email', with: 'teststuff'
+
+      find_link(t('forms.buttons.continue')).trigger('click')
+
+      expect(find_field('Email').value).to be_blank
+      expect(current_path).to eq new_user_registration_path
+    end
+
```

Comment:
Good point. Adding `expect(find_field('Email').value).to be_blank` sounds good to me.